documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,136 @@
+== Technical changes
+
+This release has the following technical changes:
+
+.Faster disk image migration without using virt-v2v
+
+IDisk images are not converted anymore using virt-v2v when migrating from RHV. This way, the migration is faster and not limited to guest operating systems that are supported by virt-v2v. link:https://github.com/kubev2v/forklift-controller/issues/403[(forklift-controller#403)]
+
+.Faster migration using conversion pod disk transfer
+
+vSphere to the local OCP cluster migration uses conversion pod instead of Containerized Data Importer (CDI), accelerating the migration.
+
+.Migrated virtual machines are not scheduled on the target OCP cluster
+
+The migrated virtual machines are no longer scheduled on the target OCP cluster. This enables migrating VMs that cannot start due to limit constraints at the migration time on the target.
+
+.StorageProfile resource needs to be updated for a non-provisioner storage class
+
+You must update the `StorageProfile` resource with `accessModes` and `volumeMode` for non-provisioner storage classes such as NFS. The documentation includes a link to the relevant procedure.
+
+[id="new-features-and-enhancements-24_{context}"]
+== New features and enhancements
+
+This release has the following features and improvements:
+
+.OCP console plugin
+
+This version of {project-full} has replaced the stand-alone UI with a UI that is integrated in the OCP Console. The new UI operates as an OCP Console plugin. It is implemented by default in version 2.4, disabling the old UI. You can enable the old UI by setting `feature_ui: true` in ForkliftController. link:https://issues.redhat.com/browse/MTV-427[(MTV-427)]
+
+.OpenStack migration
+
+See new link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#rhv-prerequisites_mtv[prerequisites] for RHV and VMware migration, and link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.3/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#migrating-virtual-machines-cli_mtv[procedure] to migrate virtual machines. link:https://issues.redhat.com/browse/MTV-426[(MTV-426)]
+
+.Skip certification option
+
+Skip certificate validation' option was added to VMware and RHV providers. If checked, the provider's certificate will not be validated and the UI will not ask for specifying a CA certificate.
+
+.Only third-party certificate required
+
+When defining a RHV provider that set with third-party certificate, only the third-party certificate needs to be specified and MTV would retrieve the Manager CA certificate automatically.
+
+.Conversion of VMs with RHEL9 guest operating system
+
+VM conversion with RHEL9 in virt-v2v installs `qemu-ga` within the converted guest. link:https://issues.redhat.com/browse/MTV-332[(MTV-332)]
+
+[id="known-issues-24_{context}"]
+== Known issues
```

Comment:
we need to add a new one:
```
.VMs from vSphere with RHEL 9 guest operating system may start with network interfaces that are down

When migrating VMs that are installed with RHEL 9 as guest operating system from vSphere, their network interfaces could be disabled when they start in Openshift Virtualization
```

Reply:
> we need to add a new one:
> 
> ```
> .VMs from vSphere with RHEL 9 guest operating system may start with network interfaces that are down
> 
> When migrating VMs that are installed with RHEL 9 as guest operating system from vSphere, their network interfaces could be disabled when they start in Openshift Virtualization
> ```

Hey Arik, do you have a ticket number for this please?

Reply:
checking with Nisim